src/WebIDL.js

```diff
@@ -4,6 +4,6 @@
 
 exports.parseImpl = function(s) {
     return function() {
-        return require('webidl2').parse(s);
+        return require('../../bower_components/webidl2').parse(s);
```

Comment:
One issue here is that we can't necessarily rely on this file being in this location relative to the current file. That's why we usually refer to PS modules with relative paths, and JS modules by name.